wordsmith/configuration.py

```diff
@@ -4,30 +4,40 @@
 
 This module implements the Wordsmith configuration object.
 """
-from . import __version__
 
-class Configuration(object):
-  """
-    Constructs a :class:`Wordsmith <Configuration>` object.
 
-    :param api_key: API key from Wordsmith.
-    :param base_url: (optional) String representing the base URL for the Wordsmith API per documentation at http://wordsmith.readme.io/v1/docs
-    :param user_agent: (optional) String representing the user agent that should be sent with each API request
+DEFAULT_VERSION = '1'
+DEFAULT_URL = 'https://api.automatedinsights.com/v' + DEFAULT_VERSION
```

Comment:
I don't think users should be allowed to change the default url; just the version. We don't really want people using this on incompatible sites and I don't think we want to support this for staging/qa/locally either.